snapcraft.yaml

```diff
@@ -41,47 +39,20 @@ apps:
       - removable-media
       - screen-inhibit-control
       - u2f-devices
-      - unity7
       - upower-observe
-      - wayland
-      - x11
     slots:
       - mpris
 plugs:
   browser-sandbox:
     interface: browser-support
     allow-sandbox: true
-  gtk-3-themes:
-    interface: content
-    target: $SNAP/data-dir/themes
-    default-provider: gtk-common-themes
-  icon-themes:
-    interface: content
-    target: $SNAP/data-dir/icons
-    default-provider: gtk-common-themes
-  sound-themes:
-    interface: content
-    target: $SNAP/data-dir/sounds
-    default-provider: gtk-common-themes
 parts:
   brave:
     plugin: dump
     source: https://github.com/brave/brave-browser/releases/download/v$SNAPCRAFT_PROJECT_VERSION/brave-browser_$SNAPCRAFT_PROJECT_VERSION_$SNAPCRAFT_TARGET_ARCH.deb
```

Comment:
Idk how this is parsed, but in many languages this would refer to a variable called `$SNAPCRAFT_PROJECT_VERSION_` (notice the trailing `_`), because `_` is a valid character in a variable name. In bash, the way to solve it would be to use the curly brace notation `${SNAPCRAFT_PROJECT_VERSION}`.

Reply:
I think in this context for the yaml snapcraft supports either syntax.  I can change it if you like, but this is a pretty common pattern we use in lots of snaps.